content/en/observability_pipelines/sources/splunk_hec.md

```diff
@@ -21,4 +23,26 @@ Select and set up this source when you [set up a pipeline][1]. The information b
 
 {{% observability_pipelines/log_source_configuration/splunk_hec %}}
 
+## Send logs from the Splunk Distribution of the OpenTelemetry Collector to Observability Pipelines
+
+To send logs from the Splunk Distribution of the OpenTelemetry Collector:
+
+1. Install the Splunk OpenTelemetry Collector based on the your environment:
+    - [Kubernetes][2]
+    - [Linux][3]
+2. Configure the Splunk OpenTelemetry Collector:
+    ```bash
+    cp /etc/otel/collector/splunk-otel-collector.conf.example etc/otel/collector/splunk-otel-collector.conf
+    ```
+    ```bash
+    # Splunk HEC endpoint URL, if forwarding to Splunk Observability Cloud
+    # SPLUNK_HEC_URL=https://ingest.us0.signalfx.com/v1/log
+    # If you're forwarding to a Splunk Enterprise instance running on example.com, with HEC at port 8088:
+    SPLUNK_HEC_URL=http://0.0.0.0:8088/services/collector
```

Comment:
We should replace this with placeholder text. `0.0.0.0` will never work here. It should be the actual addressable IP address or Load balancer URL for OP(s). So something like where we've done this elsewhere in the docs like `<IP_ADDRESS_OR_LOAD_BALANCER_URL_FOR_OP>` -- though that's a bit of a mouthful.